diff --git meta/recipes-extended/cronie/cronie/crontab meta/recipes-extended/cronie/cronie/crontab
index 22c4feb..44339a3 100644
--- meta/recipes-extended/cronie/cronie/crontab
+++ meta/recipes-extended/cronie/cronie/crontab
@@ -12,3 +12,5 @@ PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 # 30  7     * * *    root        cd / && run-parts /etc/cron.daily
 # 42  7     * * 7    root        cd / && run-parts /etc/cron.weekly
 # 55  7     1 * *    root        cd / && run-parts /etc/cron.monthly
+
+@reboot root systemctl restart systemd-networkd
